modules/rostests/apitests/gdi32/CreateIconIndirect.c

```diff
@@ -147,45 +145,45 @@ START_TEST(CreateIconIndirect)
     ok(hcursor != 0, "should not fail\n");
     ok(GetIconInfo(hcursor, &iconinfo2), "\n");
     ok(iconinfo2.fIcon == 0, "\n");
-    ok(iconinfo2.xHotspot == 0, "%ld\n", iconinfo2.xHotspot);
-    ok(iconinfo2.yHotspot == 8, "%ld\n", iconinfo2.yHotspot);
+    ok_int(iconinfo2.xHotspot, 10);
+    ok_int(iconinfo2.yHotspot, 10);
```

Comment:
It seems silly to even test this. It depends on whatever your system's cursor is set to, doesn't it?

I'd argue that's true for almost everything between L146/144 to L171/169.

Reply:
I'll improve it a bit. Most of that stuff shouldn't be here in the first place, but fixing that is not part of this PR.